### ISSUE TITLE
Overcooked!2 not compatible with Harmony 2.3

### ISSUE THREAD

**Issue by yuxuanchiadm** [newman55/unity-mod-manager#144]
Add `<HarmonyVersion>2.2</HarmonyVersion>` seems to work. Fix recent issue report in https://www.nexusmods.com/site/mods/21?tab=bugs

**Comment by newman55** [comment]
I tested harmony 2.3 on build 231.807171. I would like to clarify your version.

**Comment by yuxuanchiadm** [comment]
Game version? 1034.799570

**Comment by newman55** [comment]
Sorry, in GOG it's named 6.231(58622)

**Comment by yuxuanchiadm** [comment]
Oh. I'm on steam version of the game. Also here's my output_log.txt file:
```
Initialize engine version: 2017.4.8f1 (5ab7f4878ef1)
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce RTX 4090 (ID=0x2684)
    Vendor:   NVIDIA
    VRAM:     3072 MB
    Driver:   31.0.15.5161
Begin MonoManager ReloadAssembly


[Manager] Injection...
[Manager] Initialize.
[Manager] Version: 0.27.14.0.
[Manager] OS: Microsoft Windows NT 10.0.22631.0 x86.
[Manager] Net Framework: 2.0.50727.1433.
[Manager] Unity Engine: 2017.4.81.
[Manager] Game: Overcooked! 2.
[Manager] IsSupportOnSessionStart: False.
[Manager] IsSupportOnSessionStop: False.
[Manager] Mods path: E:\Program Files (x86)\Steam\steamapps\common\Overcooked! 2\Mods.
Fallback handler could not load library E:/Program Files (x86)/Steam/steamapps/common/Overcooked! 2/Overcooked2_Data/Mono/data-1C0C2758.dll
Fallback handler could not load library E:/Program Files (x86)/Steam/steamapps/common/Overcooked! 2/Overcooked2_Data/Mono/data-1C143010.dll
[Manager] Injection successful.
- Completed reload, in  0.185 seconds
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.404600 ms
[Manager] Starting.
[Manager] Start parsing game version.
[Manager] Found game version string: '1034.799570'.
[Manager] Game version detected as '1034.799570'.
[Manager] Parsing mods.
[Manager] Reading file 'E:\Program Files (x86)\Steam\steamapps\common\Overcooked! 2\Mods\HostForceTheme\Info.json'.
[Manager] Reading file 'E:\Program Files (x86)\Steam\steamapps\common\Overcooked! 2\Mods\RuntimeUnityEditor\Info.json'.
[Manager] Sorting mods.
[Manager] Loading mods.
```

**Comment by newman55** [comment]
Your version of the unity engine seems to be old

`[Manager] Injection...
[Manager] Initialize.
[Manager] Version: 0.27.14.0.
[Manager] OS: Microsoft Windows NT 10.0.19045.0 AMD64.
[Manager] Net Framework: 4.0.30319.42000.
[Manager] Unity Engine: 2018.4.321.
[Manager] Game: Overcooked! 2.
[Manager] IsSupportOnSessionStart: False.
[Manager] IsSupportOnSessionStop: False.
[Manager] Mods path: F:\Games\Overcooked! 2\Mods.
[Manager] Starting.
[Manager] Start parsing game version.
[Manager] Found game version string: '231.807171'.
[Manager] Game version detected as '231.807171'.
[Manager] Parsing mods.
[Manager] FINISH. SUCCESSFUL LOADED 0/0 MODS.`

**Comment by yuxuanchiadm** [comment]
So should the solution be separate Steam and GoG of the game in `UnityModManagerConfig.xml` file?

**Comment by newman55** [comment]
Is this the latest version for steam? Is there no beta?

**Comment by yuxuanchiadm** [comment]
Yes. It's latest version for steam

**Comment by newman55** [comment]
Okay, I'll fix it.

**Comment by yuxuanchiadm** [comment]
Close as issue fixed in UMM 0.27.14b
